data/mods/Xedra_Evolved/overmap/mutable_special.json

```diff
@@ -0,0 +1,169 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "Salamander eruption",
+    "subtype": "mutable",
+    "locations": [ "subterranean_empty", "land", "open_air" ],
+    "city_distance": [ 1, 40 ],
+    "occurrences": [ 80, 100 ],
+    "flags": [ "UNIQUE", "WILDERNESS" ],
+    "spawns": { "group": "GROUP_SALAMANDER_ERUPTION_SURFACE", "population": [ 100, 300 ], "radius": [ 10, 30 ] },
+    "check_for_locations": [
+      [ [ 0, 0, 0 ], [ "land" ] ],
+      [ [ 0, 0, 1 ], [ "open_air" ] ],
+      [ [ 0, 1, -1 ], [ "subterranean_empty" ] ],
+      [ [ 1, 1, -1 ], [ "subterranean_empty" ] ],
+      [ [ 0, 1, -2 ], [ "subterranean_empty" ] ],
+      [ [ 1, 1, -2 ], [ "subterranean_empty" ] ],
+      [ [ -1, 1, -2 ], [ "subterranean_empty" ] ],
+      [ [ -2, 1, -2 ], [ "subterranean_empty" ] ]
+    ],
+    "joins": [
+      "eruption_ground_to_eruption_roof",
+      "eruption_ground_to_eruption_entrance",
+      "eruption_entrance_to_tunnels",
+      "tunnel_to_tunnel",
+      "-2_to_-3",
+      "-3_tunnel_to_-3_tunnel",
+      "-3_to_-4",
+      "-4_tunnel_to_-4_tunnel",
+      "-4_tunnel_to_finale"
+    ],
+    "overmaps": {
+      "eruption_entrance": { "overmap": "salamander_eruption_ground_north", "below": "eruption_entrance_to_tunnels" },
```

Comment:
Ok, try adding this on the next line.

`"salamander_eruption_entrance": { "overmap": "salamander_eruption_entrance" },`